src/common/POLFields/EresourcesFields/FieldURL.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import { TextField } from '@folio/stripes/components';
+import { validateURL } from '@folio/stripes-acq-components';
+
+const FieldURL = () => {
```

Comment:
IMHO, it's better to pass props here and let them to be more important if there is a need to customize field

Reply:
added default props: `disabled`, `required`